.common-ci.yml

```diff
@@ -19,7 +19,7 @@ default:
 
 variables:
   GIT_SUBMODULE_STRATEGY: recursive
-  BUILDIMAGE: "${CI_REGISTRY_IMAGE}/build:${CI_COMMIT_SHORT_SHA}"
+  BUILDIMAGE: "ghcr.io/nvidia/k8s-test-infra:devel-go1.20.5"
```

Comment:
I would rather remove this entirely. See https://github.com/NVIDIA/mig-parted/pull/36.

Note that strictly speaking this depends on the golang version defined in versions.mk and extracting that here is overly complicated. We also not need the additional jobs.